docs/designs/link-output.md

```diff
@@ -57,20 +60,26 @@ Example:
         {
             "source_url": "/dotnet/articles/a",
             "source_moniker_group": "218c13d0",
-            "target_url": "/dotnet/articles/b"
+            "target_url": "/dotnet/articles/b",
+            "source_file_url": "https://github.com/dotnet/articles/a.md",
+            "source_line": "10"
         },
         {
             "source_url": "/dotnet/articles/a",
             "source_moniker_group": "218c13d0",
-            "target_url": "https://github.com"
+            "target_url": "https://github.com",
+            "source_file_url": "https://github.com/dotnet/articles/a.md",
+            "source_line": "20"
         },
         {
             "source_url": "/dotnet/articles/a",
             "source_moniker_group": "218c13d0",
-            "target_url": "./c.md"
+            "target_url": "./c.md",
+            "source_file_url": "https://github.com/dotnet/articles/a.md",
+            "source_line": "30"
         }
     ]
 }
 ```
-> Notice: `source_url` should be the root file for inclusion (TOC inclusion and markdown inclusion)
+> Notice: `source_url` and `source_file_url` should be the root file for inclusion (TOC inclusion and markdown inclusion)
```

Comment:
is it possible to point to inclusion file's github url for the links belongs to inclusion?

Reply:
you are right, will update the note